### ISSUE TITLE
configuration help

### ISSUE THREAD

**Issue by kanthgithub** [hanchchch/nestjs-slack-listener#263]
## Question
how should we configure these in .env?
the name of env variables?
where are these being injected ?
please help to add a sample.

```
[Event and Interactivity Subscription](https://github.com/hanchchch/nestjs-slack-listener#event-and-interactivity-subscription)
You need to set event and interactivity subscriptions URL of your slack app so that the app can receive events and interactivity from slack.

Event subscription
https://api.slack.com/apps/your-app-id/event-subscriptions
http://<hostname>/slack/events
Interactivity subscription
https://api.slack.com/apps/your-app-id/interactive-messages
http://<hostname>/slack/interactivity
```

**Comment by kanthgithub** [comment]
@hanchchch please help to check

**Comment by hanchchch** [comment]
Those are not env vars or something you need to inject in your code-side. You need to configure them on the slack apps setting page: https://api.slack.com

Please refer to:
1. https://api.slack.com/apis/connections/events-api#subscribing
2. https://api.slack.com/interactivity/handling#setup

**Comment by kanthgithub** [comment]
got it thanks for quick response @hanchchch 
i just created app, enabled event subscriptions and entered my API URL
would it be only one URL on which it would send any event activity?
and we gotta filter them as provided in your framework?

**Comment by hanchchch** [comment]
Yes, that's why I made this framework. I didn't want to put everything in one endpoint.

**Comment by sabahmax-dev** [comment]
Hi! Thanks for raising this. I opened [#266](https://github.com/hanchchch/nestjs-slack-listener/pull/266), which fixes the configuration confusion in this issue by adding a `.env` sample, clarifying the environment variable names, and explaining where the Slack event and interactivity URLs are injected/configured.
